### PR TITLE
changing metric names across files 

### PR DESCRIPTION
**Pull request by leslie-kim**
I can also change the function names in object_based.py, gender_based.py, etc to use these keywords. That would also change the README because the function names are listed there as well.

### CODE REVIEW

File: Geography Analysis.ipynb [hunk 1]
```diff
@@ -1557,9 +1559,9 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.7.3"
+   "version": "3.8.3"
```
Comment (Angelina-Wang) [inline Geography Analysis.ipynb]: please change back to 7

File: Gender Analysis.ipynb [hunk 2]
```diff
@@ -57,7 +57,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 20,
```
Comment (Angelina-Wang) [inline Gender Analysis.ipynb]: please clear all output cells before pushing

File: Geography Analysis.ipynb [hunk 3]
```diff
@@ -914,10 +914,12 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
+   "metadata": {
```
Comment (aliu22) [inline Geography Analysis.ipynb:917]: could this be removed by clearing metadata?

File: Object Analysis.ipynb [hunk 4]
```diff
@@ -284,36 +284,20 @@
     "to_write = {}"
    ]
   },
-  {
-   "cell_type": "code",
```
Comment (aliu22) [inline Object Analysis.ipynb]: confirming this deletion was intended

**Comment by aliu22** [comment]
Not specifically code related, but should we also change the filenames [linked](https://drive.google.com/drive/folders/1cGUr2ruV7IRl4h8EGtCjRCsg8wtPVu5P) in the drive under step 0.5 to the descriptive ones?

**Comment by Angelina-Wang** [comment]
> Not specifically code related, but should we also change the filenames [linked](https://drive.google.com/drive/folders/1cGUr2ruV7IRl4h8EGtCjRCsg8wtPVu5P) in the drive under step 0.5 to the descriptive ones?

great point! I'll do that after this PR is merged

File: Object Analysis.ipynb [hunk 5]
```diff
@@ -284,6 +284,22 @@
     "to_write = {}"
    ]
   },
+  {
```
Comment (Angelina-Wang) [inline Object Analysis.ipynb]: wait what is this cell? also 0.pkl -> new name

Comment (leslie-kim) [reply Object Analysis.ipynb]: I forgot I had deleted this cell when I was running locally bc it gave me issues with CocoDatasetNoImages, but I just added it back.

**Pull request merged**